src/quicr_client_raw_session.h

```diff
@@ -0,0 +1,251 @@
+/*
+ *  quicr_client_raw_session.h
+ *
+ *  Copyright (C) 2023
+ *  Cisco Systems, Inc.
+ *  All Rights Reserved
+ *
+ *  Description:
+ *      This file defines a session layer between the client APIs and the
+ *      transport that uses raw data packets (either UDP or QUIC).
+ *
+ *  Portability Issues:
+ *      None.
+ */
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
+
+#include "quicr/encode.h"
+#include "quicr/message_buffer.h"
+#include "quicr/quicr_common.h"
+#include "quicr/quicr_name.h"
+#include "quicr/quicr_namespace.h"
+#include "transport/transport.h"
+#include "quicr/quicr_client_delegate.h"
+#include "quicr/quicr_client.h"
+
+namespace quicr {
+
+/**
+ *   Client Raw Session Interface
+ */
+class QuicRClientRawSession : public QuicRClientSession
+{
```

Comment:
may be rename it to QuicRClientNativeSession ?

Reply:
What is Native?  These are raw packets on the wire, as opposed to being wrapped in HTTP/3 or something.

Reply:
I think Native sounds misleading, since it is usually used to refer to something in relation to an environment more than a mode.